package.json

```diff
@@ -42,10 +42,18 @@
     "node": ">=8"
   },
   "dependencies": {
-    "@babel/runtime": "^7.1.2",
     "@blueprintjs/icons": "^3.2.0",
+    "@types/arrify": "^1.0.4",
+    "@types/classnames": "^2.2.7",
+    "@types/dom-helpers": "^3.4.1",
+    "@types/fuzzaldrin-plus": "^0.6.0",
+    "@types/react": "^16.8.10",
+    "@types/react-dom": "^16.8.3",
+    "@types/react-transition-group": "^2.0.16",
+    "@types/tinycolor2": "^1.4.1",
     "arrify": "^1.0.1",
     "classnames": "^2.2.6",
+    "csstype": "^2.6.3",
```

Comment:
Should all of these `@types/*` and `csstype` be in dev dependencies?

Reply:
I believe csstype is only imported because of the ui-box types being taken internally *for now*

Reply:
It sounds like any types relied on in the output declaration file should be regular dependencies instead of dev deps. Ignore my previous comment.

Reply:
Yep they should! You'd want that for your IDE's intellisense